### PR TITLE
388/url determines assessment

### LINKED ISSUE #349: Allow multiple active assessments
old-vs-new-app-page

In the vue app page, it should be possible to:
1. See more than one assessment in progress (See 'updates' field in `/api/<organization>/benchmark/historical/` API).
2. Create a new assessment unrelated to the ones currently in progress.

### PR DESCRIPTION
**Pull request by quiaro**
Fixes #349 and #388

### CODE REVIEW

File: clients/web/src/mocks/scenarios/org_assessment_multiple.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const current_construction = server.create('assessment', {
+    account: orgId,
+    industryName: 'Construction',
+    industryPath: '/construction/',
+  })
+
+  const current_enclosures = server.create('assessment', {
+    account: orgId,
+    industryName: 'Boxes & enclosures',
+    industryPath: '/metal/boxes-and-enclosures/',
```
Comment (smirolo) [inline clients/web/src/mocks/scenarios/org_assessment_multiple.js:17]: On the API side, there is no trailing '/' in paths (ex: `/metal/boxes-and-enclosures`)

File: clients/web/src/router.js [hunk 2]
```diff
@@ -25,63 +26,70 @@ export default new Router({
...
-      path: '/:org/assess/:id/intro/*/',
+      path: '/:org/assess/:slug/intro/*/',
       name: 'introPractices',
       component: IntroCurrentPractices,
       props: true,
       pathToRegexpOptions: { strict: true },
     },
     {
-      path: '/:org/assess/:id/content/*/',
+      path: '/:org/assess/:slug/*/',
```
Comment (smirolo) [inline clients/web/src/router.js:50]: Why are you removing the `/content/` in URLs?

Comment (quiaro) [reply clients/web/src/router.js]: I reverted this change.

File: clients/web/src/mocks/scenarios/org_assessment_multiple.js (hunk 1 above)
Comment (quiaro) [reply clients/web/src/mocks/scenarios/org_assessment_multiple.js:17]: 👌 Must have been thrown off by some of the sample responses e.g. https://www.tspproject.org/docs/api/#CreatePageElementEditableDetail. Ok, I'll change it. Does this mean that *all* API calls do not have a trailing slash?

Comment (quiaro) [reply clients/web/src/mocks/scenarios/org_assessment_multiple.js:17]: Yes, per our meeting, the trailing slash is not necessary in API calls. I will create card for this.

**Pull request merged**